HULKs/hulk
# Add RANSAC lines to line detection debug panel

## Introduced Changes

What it says

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

Add more visualizations.

## How to Test

Open line detection overlay in Twix with connected robot and observe yellow lines.
